contrib/babelfishpg_tsql/src/pltsql_utils.c

```diff
@@ -1337,3 +1338,49 @@ tsql_get_trigger_rel_oid(Oid object_id)
 	table_close(tgrel, AccessShareLock);
 	return tgrelid;
 }
+
+/*
+ * Helper function to execute a utility command using
+ * ProcessUtility(). Caller should make sure their
+ * inputs are sanitized to prevent unexpected behaviour.
+ */
+void
+exec_utility_cmd_helper(char *query_str)
```

Comment:
can we see if there is already a function doing similar things, we do execute SQLs for sp_rename etc.